client/src/app/all-projects/all-projects.component.spec.ts

```diff
@@ -0,0 +1,46 @@
+import {async, ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing';
+
+import { AllProjectsComponent } from './all-projects.component';
+import {IngestService} from '../shared/services/ingest.service';
+
+describe('AllProjectsComponent', () => {
+  let component: AllProjectsComponent;
+  let fixture: ComponentFixture<AllProjectsComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ AllProjectsComponent],
+      providers: [ { provide: IngestService, useClass: MockIngestService } ]
+    })
+      .compileComponents();
+  }));
+
+  beforeEach(() => {
```

Comment:
Why 2 `beforeEach` parts?

Reply:
One is async. This is the standard boilerplate